trap_test.go

```diff
@@ -63,7 +65,7 @@ SANITY:
 		Default.SecurityParameters = test.out.SecurityParameters.Copy()
 		Default.Version = Version3
 		var buf = test.in()
-		var res = Default.UnmarshalTrap(buf, true)
+		res, _ := Default.UnmarshalTrap(buf, true)
```

Comment:
Shall we log the error ?

Reply:
Adding an assert `require.NoError(t, err)`